src/fetcher/index.js

```diff
@@ -3,9 +3,12 @@ import HttpProxyAgent from 'http-proxy-agent';
 import HttpsProxyAgent from 'https-proxy-agent';
 
 const LANGUAGE = 'en';
+const TIMEOUT = 30 * 1000;
 
 export default async function fetch(url) {
-  const options = {};
+  const options = {
+    timeout: TIMEOUT ,
```

Comment:
[Note that this option will be removed in the next version](https://github.com/node-fetch/node-fetch/blob/d351058caeccc08bbab4380d6ced987db5acf903/docs/v3-UPGRADE-GUIDE.md#the-timeout-option-was-removed).

Reply:
Yes, I had seen that. I find the added value of an app-level timeout enough to use it now. We'll use the suggested upgrade path when / if we upgrade that dependency 🙂